frontend/components.js

```diff
@@ -758,13 +760,25 @@ const NavbarAuth = connect((state) => ({
           onClick: () => window.history.pushState({}, 'dashboard', 'dashboard'),
         },
         h('img', {src: '/assets/logo.png'}),
-        h('span', undefined, 'CardanoLite Wallet'),
+        h('span', undefined, 'CardanoLite'),
+
         h('sup', undefined, '⍺')
       ),
       h(
-        'label',
-        {class: 'navcollapse-label', for: 'navcollapse'},
-        h('a', {class: 'menu-btn'}, 'Menu')
+        'div',
+        {class: 'on-mobile-only'},
```

Comment:
I would merge this code with LoginStatus component. I believe that some css will do it after that merge.